### DISCUSSION TITLE
Change Retry interval

### DISCUSSION THREAD

**Discussion by saporus** [postalserver/postal#1481] [Feature suggestions]
Hello @willpower232 ,

I trust you are doing great.

Could you please show me how to configure this 

> This message will be retried automatically in xxx minutes.

I would love to configure postal retry interval to every 10 minutes instead of postal increasing the interval.

I want to change it to a stable interval time like "10 minutes" every attempt should be 10 minutes interval in all till it get to the maximum retry time.

Thanks

**Comment by willpower232** [comment]
The increasing interval is a great way of not overloading a server which is currently overloaded or is currently down. Also the increasing interval means the message will be tried further into the future than 18 * 10 minutes.

That said, I believe the retry interval is calculated in this model https://github.com/atech/postal/blob/master/app/models/queued_message.rb

**Comment by saporus** [comment]
> The increasing interval is a great way of not overloading a server which is currently overloaded or is currently down. Also the increasing interval means the message will be tried further into the future than 18 * 10 minutes.
> 
> That said, I believe the retry interval is calculated in this model https://github.com/atech/postal/blob/master/app/models/queued_message.rb

Thanks for the reply.

But still, I am yet to know what to change in that model to be able to solve my request.
Which line exactly am I changing?
What am I going to add and how to add it?

Looking forward to read back from you.

Thanks

**Comment by willpower232** [comment]
There is literally a method in that file called `calculate_retry_time` which should be the focus of your change but what you want to change it to, I'm not sure. You should consult a ruby developer if possible.

Also be aware that any changes would be overwritten by future updates.

**Reply by arshiyaaara** [reply]
@willpower232  hey I can't find that method in that file . Can you pls help how to change the retry_time because currently its 5 min for me and I want to change it to 20 mins atleast so my IP won't be affected.

**Comment by saporus** [comment]
@catphish I trust you are doing great. I am sorry for the tag but could you please assist on my request?
MTA like Port25 has this setting "retry-after	10m" so  I would love to configure postal retry interval to every 10 minutes instead of postal increasing the interval.

I want to change it to a stable interval time like "10 minutes" every attempt should be 10 minutes interval in all till it get to the maximum retry time.

Thanks

**Comment by adamcooke** [comment]
We don't have any immediate plans to change this but I shall categorise this as a feature request.
